src/main/webapp/index.html

```diff
@@ -16,30 +16,12 @@ <h1 class="logo">smallcity</h1>
     </header>
 
     <div class="main-container">
-      <div class="map-container">
-      </div>
-      <div class="results-container">
-        <div class="results-card">
-          <div class="results-image">
-            <!-- Image of buisiness goes here via JS-->
-          </div>
-          <div class="results-business-description">
-            <h2>Business Name</h2>
-            <p>Business address</p>
-            <p>Vicinity</p>
-          </div>
-          <div class="results-rating"></div>
-          <button class="results-website-button">Visit Website</button>
-        </div>
-        <div class="results-card">
-          
-        </div>
-        <div class="results-card">
-          
-        </div>
+      <div class="map-container"></div>
+      <div class="results-container" id="results-container">
+       <div class="results-content" id="results-content"></div>
         <div class="pagination-container">
-          <button class="pagination-buttons">Previous</button>
-          <button class="pagination-buttons">Next</button>
+          <input type="button" class="pagination-buttons" value="Previous" onclick="limitList('previous')">
+          <input type="button" class="pagination-buttons" value="Next" onclick="limitList('next')">
```

Comment:
I'd suggest refactoring this into two methods: navigatePrevious and moveNext or something similar. That makes things a bit clearer, and will remove the need for some of if statements below.

Reply:
That's a great idea. I did it this way originally to avoid confusion but ended up making it more complex haha - changed